configs/opencv-ovms/demos/classification/python/classification_demo.py

```diff
@@ -255,7 +256,12 @@ def main():
         if async_pipeline.is_ready():
             # Get new image/frame
             start_time = perf_counter()
-            frame = cap.read()
+            status,frame = cap.read()
+            if not(status):
+                st = time.time()
+                cap = open_images_capture(args.input, args.loop)
+                print("time lost due to reinitialization : ",time.time()-st)
+                continue
```

Comment:
will this potentially loops forever if somehow never captured?  probably good idea to set some configurable max wait time?

Reply:
I added a retry logic